extension/content.css

```diff
@@ -605,3 +605,9 @@ div.inline-comment-form .form-actions,
 #refined-github-project-new-link {
 	margin-top: 5px;
 }
+
+/* Display notification bell icon for bookmarked Issues/PR */
+.notification-indicator .mail-status.local-unread{
+	display: inline-block;
```

Comment:
Better to do this in JS IMO (via [`$(...).show()`](https://api.jquery.com/show/))